code/modules/reagents/chemistry/reagents/misc_reagents.dm

```diff
@@ -225,7 +225,6 @@
 	name = "Burned Cooking Oil"
 	description = "It's full of char and mixed with so much crud it's probably useless."
 	id = "cooking_oil_inert"
-	reagent_after_burning = null
```

Comment:
Please also put in the PR somewhere that this also makes ash now!

Reply:
It doesn't, its entire proc is overridden to do nothing; just figured I'd yeet this redundant assignment

Reply:
Oh yeah I see it, amazing